### PR TITLE
AN-148: Add tournament page redesigned and bugs connected to it fixed

### PR DESCRIPTION
**Pull request by ArkadiuszMin**
### Jira ticket

JIRA_LINK
[AN-148](https://agh-student.atlassian.net/browse/AN-148)
[AN-127](https://agh-student.atlassian.net/browse/AN-148) 
[AN-139](https://agh-student.atlassian.net/browse/AN-148)
[AN-142](https://agh-student.atlassian.net/browse/AN-148)  
### Details
Bulk fixed all bugs connected to adding tournament on frontend side

### CODE REVIEW

File: frontend/sumo-championship/src/api/tournament.ts [hunk 1]
```diff
@@ -26,7 +28,9 @@ export const addTournament = async (general: GeneralInformation, male: Categorie
         categoryIds: categories
     }
 
-    fetch(BASE_TOURNAMENT_URL + "/add", {
+    console.log(body)
```
Comment (Skalakid) [inline frontend/sumo-championship/src/api/tournament.ts:31]: remove console.log

File: frontend/sumo-championship/src/assets/icons/plus.png
Comment (Skalakid) [inline frontend/sumo-championship/src/assets/icons/plus.png:1]: Now we use Font Awesome icons please use FaIcon component with icon name FaPlus

File: frontend/sumo-championship/src/api/club.ts [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+const clubSchema = z.object({
+    id: z.number(),
+    name: z.string(),
+    nationality: z.string()
+})
+
+
+export const getClubsForCountry = (country: string): Promise<Club[]> => {
+    //TODO: add here real backend call
+    return api.get<Club[]>("v1/club/from-country", {countryName: country})()
```
[the remark this answers is not in the document]
Comment (ArkadiuszMin) [reply frontend/sumo-championship/src/api/club.ts]: removed

File: frontend/sumo-championship/src/assets/icons/plus.png
Comment (ArkadiuszMin) [reply frontend/sumo-championship/src/assets/icons/plus.png:1]: changed

**Review by Skalakid — APPROVED** [review]

**Pull request merged**